addons/knobs/src/deprecated.js

```diff
@@ -10,13 +10,27 @@ import {
   array,
   date,
   select,
+  radioButtons,
```

Comment:
Let's not change the deprecated

Reply:
I changed the deprecated back to its original state.